embark.el

```diff
@@ -692,7 +693,8 @@ first line of the documentation string; otherwise use the word
                   (if (eq action 'metadata)
                       `(metadata (category . command))
                     (complete-with-action action commands string predicate)))
-                nil t nil 'embark--prompter-history))
+                nil t nil 'embark--prompter-history
+                (and def (symbol-name def))))
```

Comment:
I think it would be better to use embark--comand-name rather than symbol-name, to allow for lambdas, simple menu items, etc.

Reply:
Ah, this is obsoleted my a later commit, forget it.